app/src/main/java/info/nightscout/androidaps/plugins/general/overview/OverviewMenus.kt

```diff
@@ -113,6 +113,9 @@ class OverviewMenus @Inject constructor(
                     if (m == CharType.PRE) insert = predictionsAvailable
                     if (m == CharType.DEVSLOPE) insert = buildHelper.isDev()
                     if (used.contains(m.ordinal)) insert = false
+                    for (g2 in g + 1 until numOfGraphs) {
```

Comment:
this is not necessary as graph menu may appear only on previously saved configurations
once you untick it, new UI will now allow selecting graph twice (line 115)

Reply:
If you select on graph1 a curve already selected on another graph, the item is hidden in menu but the curve remains selected in graphs preferences... So curve is shown in 2 subgraph but with wrong scale on first one
This is to avoid selection of all selected curves in all graphs in menu

Reply:
New UI doesn't allow on higher graphs, but if a curve is selected on for example the Graph3, you can select it again in Graph1, the item is hidden in menu for Graph 3 but the curve remains selected and shown in Overview
See screenshot on the left and middle Dev Slope on Graphs 1-3, with PR on the right, only curves not already selected are available for selection... (shorter menu but two clicks are necessary to switch a curve from one graph to another)
![GraphMenu](https://user-images.githubusercontent.com/52934600/119570960-0ecb5600-bdb1-11eb-85fb-4d0ba0148007.jpg)

We have another possibility, it's to show all curves in all graphs, and when you enable one curve on a graph, you disable it in all other graphs in graph preferences... (only one click to switch a curve from one graph to another, but longer menu)